sdk/cognitivelanguage/ai-language-text/README.md

```diff
@@ -2,7 +2,7 @@
 
 [Azure Cognitive Service for Language](https://azure.microsoft.com/services/cognitive-services/language-service/) is a cloud-based service that provides advanced natural language processing over raw text, and includes the following main features:
 
-**Note:** This SDK targets Azure Cognitive Service for Language API version 2022-05-01.
+**Note:** This SDK targets Azure Cognitive Service for Language API version 2022-10-01-preview.
 
 - Language Detection
```

Comment:
Given how long this list is, if we're going to keep putting every feature our SDK supports in the README, we should probably start making these links to bookmarks. GitHub does generate a TOC, but it's hidden and, in my experience, not a lot of people know about it. Besides, some of these READMEs get rendered in package repo sites across different languages. Bookmarks should still work there.

Reply:
/cc @kristapratico @joseharriaga @mayurid @mssfang

Reply:
By "links to bookmarks" do you mean linked to the service docs pages?

Reply:
No, e.g.,
```markdown
- [link text](#link-bookmark)

## Link Bookmark
```
Headings are automatically transformed into slugs, though with words like "a", "an", "the", etc., some renderers remove them. If that's a concern, be explicit:
```markdown
## A Link Bookmark<a name="link-bookmark"></a>
```

My worry is that with so many features - and always more - with code snippets, the README is getting so large it's harder to navigate and quite overwhelming without either breaking it up, or at least making it easier to jump to interesting sections for the reader.

Reply:
Yeah agreed, we got this feedback awhile back: https://github.com/Azure/azure-sdk-for-python/issues/19435 (from Johan IIRC). In Python, we haven't been adding a Readme example with every new feature, instead we link out specifically to the sample.

Reply:
Good points, I guess the main question here is how to present all the features on an equal footing and still be short and sweet. How does everyone feel about Python's?

Reply:
I opened https://github.com/Azure/azure-sdk-for-js/issues/23242 to track this.